src/claim_extractor/llm_extract.py

```diff
@@ -3,63 +3,187 @@
+        temperature=0,  # 0 to 1, lower means more deterministic
+        max_tokens=os.getenv("CLAUDE_MAX_TOKENS", 4096))
 
 class ClaimExtractor:
     def __init__(
-        self, 
+        self,
         llm: Optional[BaseLanguageModel] = None,
-        schema_name: str = LINKED_TRUST 
+        schema_name: str = LINKED_TRUST
     ):
         """
         Initialize claim extractor with specified schema and LLM.
-        
+
         Args:
             llm: Language model to use (ChatOpenAI, ChatAnthropic, etc). If None, uses ChatOpenAI
             schema_name: Schema identifier or path/URL to use for extraction
             temperature: Temperature setting for the LLM if creating default
         """
         self.schema, self.meta = load_schema_info(schema_name)
         self.llm = llm or default_llm()
+        
+        # Define future tense words for filtering
+        self.future_words = [
+            "will", "plans", "exploring", "aims", "hopes", "seeks", "intends",
+            "could", "would", "may", "might", "potential", "possibly",
+            "future", "upcoming", "planned", "next year", "in the future",
+            "working to", "trying to", "seeking to", "looking to"
+        ]
+        
+        # Define mission statement indicators
+        self.mission_indicators = [
+            "bringing", "providing", "ensuring", "mission", "vision",
+            "committed to", "dedicated to", "our goal", "we believe"
+        ]
+        
         self.system_template = f"""
-        You are a JSON claim extraction specialist extracting LinkedClaims (https://identity.foundation/labs-linkedclaims/).
-        Extract claims matching this schema:
+        You are a JSON claim extraction specialist. Your task is to analyze input text and identify factual claims that can be proven true or false through evidence matching the following schema:
         {self.schema}
 
-        Meta Context:
+        Meta Context for Claims:
         {self.meta}
 
-        CRITICAL REQUIREMENTS:
-        1. subject and object MUST be URIs (http://, https://, did:, urn:, etc.)
-        2. If text mentions entities without URIs, construct appropriate URIs (e.g., https://example.com/entity/John_Smith)
-        3. claim should preferably be one of: funds_for_purpose, is_vouched_for, rated, impact, same_as, validated, related_to, owns, performed (all lowercase)
-        4. howKnown values: DOCUMENT, WEB_DOCUMENT, FIRST_HAND, SECOND_HAND, DERIVED
-
         Instructions:
-        - Extract only verifiable claims from the text
-        - Subject and object must ALWAYS be valid URIs
-        - Use lowercase for claim predicates
-        - Return empty array [] if no valid claims found
-        - Never use external knowledge
+        CRITICAL: You must NEVER extract any statement containing these words:
+        - "will", "plans", "exploring", "aims", "hopes", "seeks"
+        - "could", "would", "may", "might", "potential"
```

Comment:
this is very interesting, i would love to try it, i think i don't want it in the main library but we will want it in a potential rotation of prompts to try!  i think experimenting with the prompt will be super important.  let me get in the ability to rotate the prompt and we can experiemnt with this.  I'm not sure we want to always reject future tense words, since sometimes we might be evaluating a promise with the evaluation happening later, but it is def a good option for evaluating the past reports!